e2etest/e2etest_suite_test.go

```diff
@@ -77,7 +77,7 @@ func handleFlags() {
 	flag.StringVar(&reportPath, "report-path", "/tmp/report", "the path to be used to dump test failure information")
 	flag.StringVar(&prometheusNamespace, "prometheus-namespace", "monitoring", "the namespace prometheus is running in (if running)")
 	flag.StringVar(&externalContainers, "external-containers", "", "a comma separated list of external containers names to use for the test. (valid parameters are: ibgp-single-hop / ibgp-multi-hop / ebgp-single-hop / ebgp-multi-hop)")
-	flag.StringVar(&frrImage, "frr-image", "quay.io/frrouting/frr:9.0.2", "the image to use for the external frr containers")
+	flag.StringVar(&frrImage, "frr-image", "quay.io/frrouting/frr:9.1.0", "the image to use for the external frr containers")
```

Comment:
in theory here we could go lastest version?

Reply:
I wanted to stick to the same version we ship with frr-k8s, which will change if we sort out https://github.com/metallb/frr-k8s/issues/165

Reply:
but external frr container are based their own base image which should not give NManager issues? (but ok you might want to have the hostnetwork sometimes)

Reply:
the failure is absolutely not about nmanager. We don't have network manager issues anywhere here as both the kind nodes and the gh action runners are ubuntu based.